packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -1335,26 +1586,19 @@ class _AnimatedPositionedDirectionalState extends AnimatedWidgetBaseState<Animat
 ///  * [AnimatedSlide], for animating the translation of a child by a given offset relative to its size.
 ///  * [ScaleTransition], an explicitly animated version of this widget, where
 ///    an [Animation] is provided by the caller instead of being built in.
-class AnimatedScale extends ImplicitlyAnimatedWidget {
+class AnimatedScale extends AnimatedValue<double> {
   /// Creates a widget that animates its scale implicitly.
   const AnimatedScale({
     super.key,
-    this.child,
-    required this.scale,
+    required double scale,
+    super.initialValue,
```

Comment:
initialScale? Also document this new param.

Reply:
documentation added!